__tests__/app.test.js

```diff
@@ -0,0 +1,33 @@
+const app = require("../app");
+const pool = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+const request = require("supertest");
+
+beforeEach(() => seed(testData));
+
+afterAll(() => {
+  pool.end();
+});
+
+describe("GET /api/topics endpoint", () => {
+  test("Responds with an array of topic objects with a slug and description property", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        topics = response.body;
+        //onsole.log(response.body)
```

Comment:
try to avoid leaving comments in your code as you are working, it just creates more things for you to remove later :)